.slashrc

```diff
@@ -88,6 +88,12 @@ class MediaPlugin(slash.plugins.PluginInterface):
     parser.add_argument("--parallel-metrics", action = "store_true")
     parser.add_argument("--platform", default = None, type = str, required = True)
     parser.add_argument("--device", default = '/dev/dri/renderD128', type = str, help = 'vaapi-fits run with render device')
+    parser.add_argument("--stringapi", default = 0,
+      type = int, help = (
+        f"{int(0)}   = default, disabled string api; "
+        f"{int(1)}   = only enabled string api; "
+        f"{int(2)}   = random call parameters with enabling and disabling string api"
+        " (default: %(default)i)"))
```

Comment:
Hmm... this option is specific to ffmpeg-qsv only and I don't like exposing it at the framework core.  We should think of a better way.  Maybe just create string api specific test functions in ffmpeg-qsv??? IDK.

Reply:
> Hmm... this option is specific to ffmpeg-qsv only and I don't like exposing it at the framework core. We should think of a better way. Maybe just create string api specific test functions in ffmpeg-qsv??? IDK.
hi Artie, gst-libvpl will enable soon. Mengke is on developping gstreamer source codes.

Reply:
Right.  And I don't think we should do that.  I think they should be new tests that are added to current test context without requiring us to change the CI test command line.